config/webpack.config.js

```diff
@@ -93,7 +93,7 @@ module.exports = (env = {}) => {
     scaffold: false,
     watch: false,
     setPublicPath: false,
-    destination: path.resolve(__dirname, '../', 'build', buildtype),
+    destination: buildtype,
```

Comment:
Does this change affect the `destination` used in the dev config (used for `yarn watch`)?

Reply:
@U-DON good point! I've added a fix for it in the dev config.